docker/run.sh

```diff
@@ -44,32 +45,37 @@ done
 
 echo_status "Starting up..."
 
-# Create Salt Config
-echo "file_client: local" > /etc/salt/minion
-echo "postgres.host: '${TETHYS_DB_HOST}'" >> /etc/salt/minion
-echo "postgres.port: '${TETHYS_DB_PORT}'" >> /etc/salt/minion
-echo "postgres.user: '${TETHYS_DB_USERNAME}'" >> /etc/salt/minion
-echo "postgres.pass: '${TETHYS_DB_PASSWORD}'" >> /etc/salt/minion
-echo "postgres.bins_dir: '${CONDA_HOME}/envs/${CONDA_ENV_NAME}/bin'" >> /etc/salt/minion
 
 if [[ $test = false ]]; then
   # Set extra ENVs
   export NGINX_USER=$(grep 'user .*;' /etc/nginx/nginx.conf | awk '{print $2}' | awk -F';' '{print $1}')
 
   # Apply States
   echo_status "Checking if DB is ready"
+  if [[ $db_engine == "django.db.backends.postgresql" ]]; then
```

Comment:
I think we also want this whole `if` block wrapped in another `if` block that checks if `SKIP_DB_SETUP` is `False`.

Reply:
I have added the extra if statement to handle the skip db env variable. That was good catch.
https://github.com/tethysplatform/tethys/pull/1013/commits/8ce70dd5ce57bdc488fd42ae297dfd1824476614